web/channels/player_channel.ex

```diff
@@ -34,7 +34,12 @@ defmodule Yggdrasil.PlayerChannel do
   end
 
   def handle_in("player_cmd", %{ "text" => text }, socket) do
-    Player.run_cmd(socket.assigns.user, Message.command(text))
-    {:noreply, socket}
+    case Parser.parse(text) do
```

Comment:
With this everything is wired up now. If you type `move east` you'll get the message from the command. Kind of cool to see if working. I guess I need to write some tests to cover what I can and move on to the next thing.